test-suite/squidconf/bad-acl-src-dupe.conf

```diff
@@ -0,0 +1,25 @@
+## Copyright (C) 1996-2023 The Squid Software Foundation and contributors
+##
+## Squid software is distributed under GPLv2+ license and includes
+## contributions from numerous individuals and organizations.
+## Please see the COPYING and CONTRIBUTORS files for details.
+##
+
+acl test11 src 127.0.0.1  127.0.0.0-127.0.0.255
+acl test12 src 192.168.1.0/24 192.168.0.0/16
+
+acl test13 src 127.0.0.0-127.0.0.255  127.0.0.1
+acl test14 src 127.0.0.0-127.0.0.128  127.0.0.128-127.0.0.255
+
+acl test15 src 10.0.0.1-10.0.0.128 10.0.0.0-10.0.0.1 10.0.0.128-10.0.0.255
+
+acl test25 dst 127.0.0.0-127.0.0.128/32  127.0.0.128-127.1.0.255
+
+acl test36 dst 127.0.0.1-127.0.0.128  127.0.0.0-127.1.0.0/16
+acl test37 dst 127.0.0.0-127.1.0.0/16 127.0.0.1-127.0.0.128
+
+# TODO: make configurable depending on USE_IPV6
```

Comment:
I know we have code to address this TODO. We will add that code in this PR if others insist. Otherwise, we will do a small dedicated PR after asking whether others would prefer to

* split such test files into IPv4 and IPv6 ones OR
* keep all tests together but skip (all) tests in no-IPv6 cases.